ReactNativeDeferredTemplate/android/app/src/main/AndroidManifest.xml

```diff
@@ -55,8 +55,6 @@
             </intent-filter>
         </activity>
         -->
-
-        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity"
-            android:exported="true" />
```

Comment:
@brandonpage or @wmathurin - Of the three templates, only this one was producing an inspector warning for this exported activity.  Was this needed?

Reply:
I'd not have changed it, except I was comparing all three React Native templates at the directory level to be sure I'd synchronized the updates accurately.  This was an outlier in the diff that seemed worthy of cleanup.